flask_project/campaign_manager/test/test_models.py

```diff
@@ -1,573 +1,296 @@
-    for tbl in reversed(meta.sorted_tables):
-        engine.execute(tbl.delete())
 
+def setUp():
+    """ Creates a DB for testing.
+    Will be called before every test.
+    """
+    session.commit()
+    Base.metadata.drop_all(engine)
+    Base.metadata.create_all(engine)
 
-# Create a session for the Database
-def create_session(engine):
-    Session = sessionmaker(bind=engine)
-    Session.configure(bind=engine)
-    session = Session()
-    return session
+    # create mock user
+    mock_user = User(
+            osm_user_id="user_new",
+            email="email@gmail.com"
+            )
+    mock_user.create()
 
+    # create mock campaign
+    mock_campaign = Campaign(
+            creator_id=mock_user.id,
+            name="campaign_new",
+            description="campaign_description",
+            start_date=datetime.date(2018, 1, 1),
+            end_date=datetime.date(2018, 1, 1),
+            create_on=datetime.datetime.now(),
+            uuid="mock_campaign_uuid"
+            )
+    mock_campaign.create()
 
-engine_test = db_setUp()
-Base.metadata.create_all(engine_test)
-session = create_session(engine_test)
+    # create mock feature type
+    mock_feature_type = FeatureType(
+            feature="feature",
+            name="feature_name"
+            )
+    mock_feature_type.create()
 
+    # create mock template feature
+    mock_feature_type = FeatureType(
+            feature="template",
+            name="feature_name",
+            is_template=True
+            )
+    mock_feature_type.create()
+    mock_feature_template = FeatureTemplate(
+            name="Template",
+            description="Template_description",
+            featureType_id=mock_feature_type.id
+            )
+    mock_feature_template.create()
 
-"""Tests for Table: User"""
+    # create mock taskboundary
+    geom_obj = from_shape(asShape(geom), srid=4326)
+    mock_taskBoundary = TaskBoundary(
+        coordinates=geom_obj,
+        name="name",
+        status="status",
+        type_boundary="type_boundary",
+        campaign_id=mock_campaign.id
+        )
+    mock_taskBoundary.create()
 
+    # create mock team
+    mock_team = Team(
+        name="Team7",
+        boundary_id=mock_taskBoundary.id
+        )
+    mock_team.create()
 
-def get_mock_user(username, email):
-    mock_user = User(
-        osm_user_id=username,
-        email=email
+    # create mock function
+    mock_function = Function(
+        name="Function1",
+        feature="data_collection",
+        type_id=mock_feature_type.id
         )
-    session.add(mock_user)
-    session.commit()
-    return mock_user
+    mock_function.create()
 
 
-def delete_mock_user(user_object):
-    session.delete(user_object)
-    session.commit()
+user = User()
+campaign = Campaign()
+feature_type = FeatureType()
+template_feature = FeatureTemplate()
+task_boundary = TaskBoundary()
+team = Team()
+function = Function()
 
 
-class TestUser(TestCase):
+"""Tests for Table: User"""
```

Comment:
this docstring should go under the TestUser class definition. With a full stop at the end of the sentence.